plugins/org.polymap.service.geoserver/src/org/polymap/service/geoserver/GeoServerServlet.java

```diff
@@ -177,9 +204,11 @@ public void init( ServletConfig config ) throws ServletException {
         context = new BundleServletContext( getServletContext() );
         log.info( "initGeoServer(): contextPath=" + context.getContextPath() );
         
-//        sessionKey = SessionContext.current().getSessionKey();
-//        assert sessionKey != null;
-
+        //String sessionKey = SessionContext.current().getSessionKey();
+        //assert sessionKey != null;
+        sessionKey = "wms_" + System.currentTimeMillis();
```

Comment:
oben steht "ows_" hier ist es "wms_"... das stimmt nicht, oder?

für dieses gesamte SessionContext handling gab/gibt es ServiceContext, weil es eben viel zu beachten gibt. Ich hatte ServiceContext wahrscheinlich eben genau deshalb auskommentiert: weil migration von p3 und richtig testen aufwendig sind. Außerdem ist die behandlung von entity events in den GeoServerStarter und OnDemandServlet gewandert. Und: mit dem SessionContext pro servlet instanz muss unbedingt die #map instance aus der jeweiligen session kommen.